example.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <script type="text/javascript" src="http://visjs.org/dist/vis.js"></script>
     <link href="http://visjs.org/dist/vis-network.min.css" rel="stylesheet" type="text/css" />
-    <script type="text/javascript" src="https://happy-noether-51da01.netlify.com/lib.js"></script>
+    <script type="text/javascript" src="./dist/bundle.js"></script>

```

Comment:
Because of these changes, people will have to first build everything before they can start using the library. I'd like to somehow keep it so that people can copy-paste the example and get started immediately.

The netlify thingy I've used now is really easy but not perfect, maybe you have some ideas of how to do this instead?

Reply:
Does this library get built to netlify when the code is committed to master? If it is, we could leave it the same.

I think the discussion will be a little different if you decide to publish this to npm (is that your goal?)

Reply:
Yes it does, right now it doesn't run anything yet, but I could make it run `npm build` of course. Then still the output file is now in a different location (in `dist`). I don't know if you are allowed to overwrite the current file. If so we can fix that for now.

In the end probably publishing on npm makes more sense, maybe using unpkg for the example/easy include.

Reply:
To provide some additional context - some libraries I've seen that use npm choose to instruct the user what commands they need to run in order to see the library. But I'm ok with reverting for now until we have a solution around versioning.

Reply:
OK, let's do this:
- Revert only this line, so the example will still use the old version
- I'll disable the netlify deploy for now to make sure that it keeps on working for whoever uses it
- I'll set everything up on npm, you don't have to wait for this

Reply:
Oh! I didn't realize you were online right now. ha!

I went ahead and reverted this line - I changed the .gitignore so that we check in the /build/lib.js (this should be the file that netlify should upload).

I was going to ask if you could get netlify to pick up the file from the /build folder?